tests/ui/traits/const-traits/const-drop-fail.rs

```diff
@@ -25,6 +25,8 @@ const fn check<T: ~const Destruct>(_: T) {}
 macro_rules! check_all {
     ($($exp:expr),*$(,)?) => {$(
         const _: () = check($exp);
+        //~^ ERROR the trait bound `NonTrivialDrop: const Destruct` is not satisfied
```

Comment:
The span here sucks because we point at the span of the `check($exp)` expr, rather than pointing at the argument. I plan on fixing that soon.

Reply:
Soon: https://github.com/rust-lang/rust/pull/133403